lighthouse-core/lib/web-inspector.js

```diff
@@ -239,10 +239,19 @@ module.exports = (function() {
    * Creates a new WebInspector NetworkManager using a mocked Target.
    * @return {!WebInspector.NetworkManager}
    */
-  WebInspector.NetworkManager.createWithFakeTarget = function() {
+  WebInspector.NetworkManager.createWithFakeTarget = function(driver) {
     // Mocked-up WebInspector Target for NetworkManager
     const fakeNetworkAgent = {
-      enable() {}
+      enable() {},
+      getResponseBody(requestId, onComplete) {
```

Comment:
kinda wonky that we have to do this opaquely through the `resourceContent()` instead of explicitly in the gatherer that we need it :/ @paulirish the decision was based on wanting to use NetworkRequest from devtools more aggressively right?